spec/models/concerns/digital_object_concerns/validations_spec.rb

```diff
@@ -0,0 +1,21 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+RSpec.describe DigitalObjectConcerns::Validations, type: :model do
```

Comment:
```suggestion
RSpec.describe DigitalObjectConcerns::Validations, type: :model do
  include_context 'with stubbed search adapters'
```
To allow this test to run without solr.